instrumentation/rdkafka/lib/opentelemetry/instrumentation/rdkafka/instrumentation.rb

```diff
@@ -11,7 +11,7 @@ module Rdkafka
       class Instrumentation < OpenTelemetry::Instrumentation::Base
         compatible do
           gem_version = Gem::Version.new(::Rdkafka::VERSION)
-          Gem::Requirement.new('>= 0.10.0', '< 0.20.0').satisfied_by?(gem_version)
+          Gem::Requirement.new('>= 0.10.0', '< 0.22.0').satisfied_by?(gem_version)
```

Comment:
I think @arielvalentin means to change to `Gem::Requirement.new('>= 0.20.0').satisfied_by?(gem_version)` for new instrumentation-rdkafka version